### ISSUE TITLE
Eldritch Monoliths not spawning in other dimensions than 0

### ISSUE THREAD

**Issue by Mistercl0ud** [Azanor/thaumcraft#1620]
Could you please add the option in the config file to allow the spawning of Eldritch Monoliths outside dimension 0. It's impossible to enjoy the end game part of the mod if one play on a void world or a single biome world.

**Comment by commonsensei** [comment]
OFF-TOPIC: Is there a way to force-generate thaumcraft stuff into a world? I've always wanted to play some skyblock with thaumcraft.

**Comment by TheMadmanofChaos** [comment]
This is just conjecture from someone who hasn't seen the code for this, but I believe the reason they only spawn in the overworld is due to how returning home and dungeon spawning works. It sends you to locations in both dimensions based on relative location in the world(The X, Y, and Z coordinates). If the portal was to form in a dimension other then the overworld, it would in all likelihood send you to the overworld based on the dungeons location due to the fact it doesn't know where you originally entered from(Assuming it is always the overworld, since that is the only place they  spawn,)  You also need to consider the impact dimensions would have on monolith spawning, as they would appear to become far rarer due to the possible spawns needing to be spread across all dimensions, and even then you might have two monoliths from two dimensions cross each other when forming the dungeons. With them only spawning in the overworld, Azanor is able to make a few assumptions into how things will happen, and having them spawn in other dimensions complicates the logic behind this significantly.

As for @commonsensei , I would recommend looking into the mod ex niliho and it's addons, as they do add features that makes modded skyblocks far more feasible.

**Comment by Azanor** [comment]
Yup, it is partly due to what Madman described, but that is not too hard to work around. The main reason I block them in other dims is mostly due to mods like Mystcraft that can trivialize certain types of content.
I might add a config option in 1.8 where you can set which dim you consider your "overworld" and only have stuff spawn in there.

**Comment by Mistercl0ud** [comment]
Is there a command to spawn one of them somewhere, and be able to use it ?

**Comment by MentalMouse** [comment]
There is a spawner item available in creative, it only makes the central plinth.  You need to set up the altars and caps yourself.  Also if you use one too close to an existing monolith it can dump you in the void.
